system/Database/Forge.php

```diff
@@ -513,7 +517,7 @@ public function createTable(string $table, bool $ifNotExists = false, array $att
 
             // Most databases don't support creating indexes from within the CREATE TABLE statement
             if (! empty($this->keys)) {
```

Comment:
Added second parameter for _processIndexes(). with true the method will return stand alone sql statements. More on this later.